src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -166,7 +168,7 @@ internal AuthenticationResult(MsalAccessTokenCacheItem msalAccessTokenCacheItem,
         /// </example>
         public string CreateAuthorizationHeader()
         {
-            return Oauth2AuthorizationHeader + AccessToken;
+            return _authenticationScheme?.AuthorizationHeaderPrefix ?? "Bearer" + " " + AccessToken;
```

Comment:
>Bearer [](start = 72, length = 6)

this will only and always be for a bearer token? Can you use the BearerAuthSchemePrevix value here instead "Bearer"

Reply:
why would an AuthorizationHeaderPrefix be null?

Reply:
Agree with @jmprieur here.  the _authenticationScheme can't be null (you validate that in the constructor).  So this should always be:  return _authenticationScheme.AuthorizatonHeaderPrefix;

Reply:
There is actually a ctor which we expose so that our customers can test. I can't expose _authenticationScheme there. I will expose the actual prefix though.